components/Autocomplete/index.jsx

```diff
@@ -0,0 +1,258 @@
+import React, {PropTypes} from 'react';
+import Autosuggest from 'react-autosuggest';
+import Scroll from 'react-scroll';
+
+import styles from './index.scss';
+
+const isMobile = window.outerWidth < 550;
```

Comment:
I do not think it is safe to leave this in a local file.
We may end up with many different ways to check if the user is on mobile.